app/services/item_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ItemService
+  def self.get_item_merch(item_id)
+    response = conn.get("items/#{item_id}/merchant")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.get_item(item_id)
+    response = conn.get("items/#{item_id}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.get_items
+    response = conn.get('items')
+    JSON.parse(response.body, symbolize_names: true)
+  end
+  
+  def self.conn
+    url = 'http://localhost:3000/api/v1/'
+    Faraday.new(url: url)
+  end
```

Comment:
This method is repeated in `MerchantService`. Do these need to be separate services, or could we roll them into something slightly more generic like `RailsEngineService`? It might also be interesting to have `ItemService` and `MerchantService` inherit from `RailsEngineService` to DRY up this code.